src/plugins/dashboard/config.ts

```diff
@@ -20,7 +20,7 @@
 import { schema, TypeOf } from '@kbn/config-schema';
 
 export const configSchema = schema.object({
-  allowByValueEmbeddables: schema.boolean({ defaultValue: false }),
+  allowByValueEmbeddables: schema.boolean({ defaultValue: true }),
```

Comment:
This will be reverted

Reply:
If you don't want to worry about turning this on and off, you can put 
`dashboard.allowByValueEmbeddables: true` in your `/config/kibana.dev.yml`